src/System.Memory/tests/Performance/Perf.Utf8Parser.cs

```diff
@@ -802,7 +802,59 @@ private static void ByteSpanToTimeOffsetR(string text)
             }
         }
 
-        [Benchmark(InnerIterationCount = InnerCount)]
+		[Benchmark(InnerIterationCount = InnerCount)]
+		[InlineData("1997-07-16")]
+		[InlineData("1997-07-16T19:20")]
+		[InlineData("1997-07-16T19:20:30")]
+		[InlineData("1997-07-16T19:20:30.45")]
+		[InlineData("1997-07-16T19:20+01:00")]
+		[InlineData("1997-07-16T19:20:30+01:00")]
+		[InlineData("1997-07-16T19:20:30.45+01:00")]
+		private static void ByteSpanToDateTimeJ(string text)
+		{
+			byte[] utf8ByteArray = Encoding.UTF8.GetBytes(text);
+			ReadOnlySpan<byte> utf8ByteSpan = utf8ByteArray;
+
+			foreach (BenchmarkIteration iteration in Benchmark.Iterations)
+			{
+				using (iteration.StartMeasurement())
+				{
+					for (int i = 0; i < Benchmark.InnerIterationCount; i++)
+					{
+						Utf8Parser.TryParse(utf8ByteSpan, out DateTime value, out int bytesConsumed, 'J');
```

Comment:
It would be good to see the result of a comparison using 'J' and 'O' for one of the inputs that can be parsed by both, say `1997-07-16T19:20:30.45+01:00` along with a baseline using `DateTime.TryParseExact`

Reply:
Please see https://github.com/dotnet/coreclr/pull/22999#issuecomment-470255347.